mlir/lib/Dialect/Rock/Tuning/RockTuningImpl.cpp

```diff
@@ -257,9 +257,9 @@ void createGemmTuningRangeBF(TuningParamSet *newSpace,
                                                splitKFactor, forceUnroll, true);
                     if (gemmMPerBlock >= gemmMPerWave &&
                         gemmNPerBlock >= gemmMnPerXdl) {
-                      if (kind == TuningParamSetKind::Exhaustive ||
-                          (succeeded(tuningInfo.paramsProbablyValid(
-                               b, info, gemmParams)) &&
+                      if (succeeded(tuningInfo.paramsProbablyValid(
+                              b, info, gemmParams)) &&
+                          (kind == TuningParamSetKind::Exhaustive ||
```

Comment:
However, IIRC, TuningParamSetKind::Exhaustive was introduced to make all configs run through applicability.
Now this changes that to be a no op (for this release) because couldBePerformant is empty. See : 

https://github.com/ROCm/rocMLIR/blob/5261f9fb8cc5e0109cf59bbf4cb11cc6fb527209/mlir/lib/Dialect/Rock/Tuning/GridwiseGemmParams.cpp#L215-L221

Having said that Im onboard with the change on the premise TuningParamSetKind::Exhaustive is nearing the end of its usefulness.

Reply:
This also makes it match all the other cases, which always check `paramsProbablyValid()`